.github/workflows/mobile-compile_time_options.yml

```diff
@@ -112,4 +112,14 @@ jobs:
           --define=google_grpc=disabled \
           --define=envoy_yaml=disabled \
           --@com_envoyproxy_protoc_gen_validate//bazel:template-flavor= \
-          //:android_dist
+      //:android_dist && ./bazelw test \
```

Comment:
This seems unrelated. Was it added on accident?

Reply:
sorry related, the runtime fixes were the only thing stopping us from having e2e !yaml tests for E-M. I can do that in a follow-up PR but if there were problems I'd have to roll back together so wanted to land it together.  I'll update PR description for clarity

Reply:
Makes sense, sgtm, leave it in this PR